deployment/ansible/raster-foundry.yml

```diff
@@ -32,3 +32,11 @@
     - name: Set Environment variable for host user (used to namespace jars and other artifacts in development)
       lineinfile: dest=/etc/environment regexp=^RF_HOST_USER line="RF_HOST_USER={{ host_user }}"
       when: host_user is defined
+
+    - name: Create script to handle core dumps
+      copy: content="#!/bin/sh\nexec /bin/gzip -1 > /tmp/core.$1.$2.gz\n"
```

Comment:
I left this bit in because otherwise, there isn't a clear way to know a file in `/tmp` is a core dump based on file name. In S3, they're under a core dump specific prefix.

Reply:
Gotcha gotcha. That makes sense.